src/view/trip-route-edit-point.js

```diff
@@ -73,25 +77,70 @@ const createTripRouteEditPointTemplate = (point) => {
       <section class="event__section  event__section--destination">
         <h3 class="event__section-title  event__section-title--destination">Destination</h3>
         <p class="event__destination-description">${destinationDetails.description}</p>
+        <div class="event__photos-container">
+        <div class="event__photos-tape">
+          ${destinationDetails.photos.map((photo) => `<img class="event__photo" src="${photo}" alt="Event photo">`).join('')}
+        </div>
+      </div>
       </section>
     </section>
   </form>
 </li>`;
 };
 
 
-export default class TripRouteEditPoint extends AbstractView{
+export default class TripRouteEditPoint extends SmartView{
   constructor(point) {
     super();
-    this._point = point;
+    this._data = point;
     this._element = null;
     this._editClickHandler = this._editClickHandler.bind(this);
     this._formSubmit = this._formSubmit.bind(this);
+
+    this._changePointTypeHandler = this._changePointTypeHandler.bind(this);
+    this._changeDestenationHandler = this._changeDestenationHandler.bind(this);
+
+    this._setInnerHandlers();
+  }
+
+  _setInnerHandlers() {
+    this.getElement()
+      .querySelector('.event__type-list')
+      .addEventListener('click',this._changePointTypeHandler);
+    this.getElement()
+      .querySelector('#event-destination-1')
+      .addEventListener('change',this._changeDestenationHandler);
+  }
+
+  _changePointTypeHandler (evt) {
+    if(event.target.tagName == 'INPUT'){
```

Comment:
```suggestion
    if(evt.target.tagName === 'INPUT'){
```
Или может просто другое событие использовать?